src/pineko/fonll.py

```diff
@@ -17,13 +17,24 @@
 logger = logging.getLogger(__name__)
 
 FNS_BASE_PTO = {"FONLL-A": 1, "FONLL-B": 1, "FONLL-C": 2, "FONLL-D": 2, "FONLL-E": 3}
+"""Mapping between pertubative orders as in the theory card PTO and the FONLL scheme names.
+
+The explict mapping is the following (evolution, massive parts, massless parts):
```

Comment:
Why do you need to distinguish `evolution` from `massive parts`?  From what I understood the resummation (NLL in FONLL) is just labeled by `massless parts`, so the other two have to be the same...

Reply:
It's not so much redundancy on its own, it is actually about the meaning. We have both evolution for massive and massless, and they have two different orders.

Why do you need to specify it separately? Which evolution are you referring to? (I guess the massive one, but I wonder why...)

Reply:
> We have both evolution for massive and massless, and they have two different orders.

no - they have both the same order, which indeed coincides with the massless order

Reply:
Sorry, I guess massless is asy (*massless* keeps being an ambiguous word, whether it is the *massless limit* or *zero-mass*).

But then they all should have the same order (as I argued back then), otherwise you see it's rather inconsistent - you're definitely taking a weird limit...
(you do not subtract the logs of the extra mass, because they are not in the zero-mass, but you subtract the evolution logs of all the other masses, that are also not in the zero-mass - that's quite arbitrary)